recipes/involucro/meta.yaml

```diff
@@ -3,24 +3,37 @@ package:
   version: 1.1.2
 
 source:
-  - url: https://github.com/involucro/involucro/releases/download/v1.1.2/involucro  # [linux]
-    sha256: df9bbb6490ce0e507fe735419e526140ad9567a8da85e8bf529c69505be1f8cf  # [linux]
-  - url: https://github.com/involucro/involucro/releases/download/v1.1.2/involucro.darwin  # [osx]
-    sha256: 5c50c79232f34ca439c8dbeb2c9d09998a2e90c8ee42af349a6fdb7ede7752da  # [osx]
+  - url: https://github.com/involucro/involucro/archive/v1.1.2.tar.gz
+    sha256: be8d1b65e4841bc1ae68e418aef00cbc19052038415858f4cceb44d100dbefc0
+    folder: src/github.com/involucro/involucro
+  # Add vendor-vendor sources for missing license files.
+  - url: https://github.com/mattn/go-isatty/archive/v0.0.1.tar.gz
+    sha256: 71e6931ff2804ef711eae1882fe95beee5b60c38d929c638d553ba143a3bc7ec
+    folder: src/github.com/involucro/involucro/vendor/github.com/fatih/color/vendor/github.com/mattn/go-isatty
+  - url: https://github.com/mattn/go-colorable/archive/v0.0.1.tar.gz
+    sha256: 626e0dcec24fd3621e40eceb33791747f0970c248500930a1461f599acd1aadd
+    folder: src/github.com/involucro/involucro/vendor/github.com/fatih/color/vendor/github.com/mattn/go-colorable
```

Comment:
This is purely for the `LICENSE` files and can be removed when/if involucro updates its dependencies upstream.